contracts/governance/staking/GamifiedToken.sol

```diff
@@ -501,32 +464,33 @@ abstract contract GamifiedToken is
         uint256 _rawAmount,
         uint128 _cooldownPercentage
     ) internal virtual updateReward(_account) {
-        require(_account != address(0), "ERC20: burn from the zero address");
+        require(_account != address(0), "ERC20: burn from zero address");
 
         // 1. Get and update current balance
         (Balance memory oldBalance, uint256 oldScaledBalance) = _prepareOldBalance(_account);
-        require(oldBalance.raw >= _rawAmount, "ERC20: burn amount exceeds balance");
+        require(oldBalance.raw >= _rawAmount, "ERC20: burn amount > balance");
         unchecked {
             _balances[_account].raw = oldBalance.raw - SafeCast.toUint128(_rawAmount);
         }
 
         // 2. Change the cooldown percentage based on size of recent withdrawal
-        _balances[_account].cooldownMultiplier = SafeCast.toUint16(_cooldownPercentage / 1e16);
+        _balances[_account].cooldownMultiplier = SafeCast.toUint16(_cooldownPercentage / 1e18);
```

Comment:
Are you sure this is correct? If cooldown is 50% (5e17), we want to store it as "50", so dividing by 1e16 should be right

Reply:
I've written a test for this and you are right. I'll revert